system/security/shared/SecureUser.hpp

```diff
@@ -328,6 +328,11 @@ class CSecureUser : implements ISecUser, implements ISecCredentials, public CInt
     {
         return false;
     }
+
+    virtual bool isCanonicalMatch(const char* name) const override
```

Comment:
I think a comment on what this function does is warranted. Future SecMgr develops will thank you !

Reply:
Will add the first comment to the interface, so future developers don't have to track down all implementations to figure it out.